### PR TITLE
Minor nits to documentation

### PR DESCRIPTION
**Pull request by krishvoor**

### CODE REVIEW

**Comment by krishvoor** [comment]
@morenod : very minor nit

File: hypershift/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 git-python
 ConfigParser
+elasticsearch
```
Comment (morenod) [inline hypershift/requirements.txt]: this is not required, ES module is imported on libs/requirements.txt

File: rosa-hypershift/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 git-python
 ConfigParser
+elasticsearch
```
Comment (morenod) [inline rosa-hypershift/requirements.txt]: this is not required, ES module is imported on libs/requirements.txt

**Review by morenod — CHANGES_REQUESTED** [review]
Can you please include this change on the other PR you have?

File: rosa-hypershift/requirements.txt (hunk 2 above)
Comment (krishvoor) [reply rosa-hypershift/requirements.txt]: Updated

File: hypershift/requirements.txt (hunk 1 above)
Comment (krishvoor) [reply hypershift/requirements.txt]: Updated

**Pull request closed without merging**